SecondYearFirstSemestr/SimpleFTP/SimpleFTP/Server.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace SimpleFTP
+{
+    public class Server
+    {
+        TcpListener tcpListener;
+        TcpClient? tcpClient;
+        NetworkStream? stream;
+
+        public Server(String address, int port)
+        {
+            var ip = IPAddress.Parse(address);
+            tcpListener = new TcpListener(ip, port);
+        }
+
+        public async void Start()
+        {
+            try
+            {
+                tcpListener.Start();
+            }
+            catch (Exception ex)
+            {
+                return;
+            }
+
+            while (true)
+            {
+                tcpClient = await tcpListener.AcceptTcpClientAsync();
+                Task.Run(async ()=> await ParallelRequestsProcessing(tcpClient));              
+            }
+        }
+
+        private async void ResponseToListRequest(String path)
+        {
+            if (!Directory.Exists(path))
+            {
+                await stream.WriteAsync(Encoding.UTF8.GetBytes("-1\n"));
+                return;
+            }
+
+            String[] directories = Directory.GetDirectories(path);
+            String[] files = Directory.GetFiles(path);
+            int count = directories.Length + files.Length;
+
+            var response = new StringBuilder();
+            response.Append(count + " ");
+            for (int i = 0; i < directories.Length; i++)
+            {
+                response.Append("/." + directories[i] + " false ");
+            }
```

Comment:
```suggestion
            for (var directory: directories)
            {
                response.Append("/." + directory + " false ");
            }
```
чтобы было стильно, модно, молодёжно)

Reply:
пхпх)